acceptance/services_test.go

```diff
@@ -78,6 +81,187 @@ var _ = Describe("Services", func() {
+
+			Expect(out).To(MatchRegexp(regex.TableRow(service, regex.DateRegex, "mysql-dev", "not-ready")))
+
+			By("wait for deployment")
+			Eventually(func() string {
+				out, _ := env.Epinio("", "service", "list")
+				return out
+			}, "2m", "5s").Should(MatchRegexp("deployed"))
+
+			By(fmt.Sprintf("%s/%s up", namespace, service))
+		})
+	})
+
+	Describe("ListAll", func() {
+		var namespace1, namespace2 string
+		var service1, service2 string
+		var tmpSettingsPath string
+		var user1, password1 string
+		var user2, password2 string
+
+		updateSettings := func(user, password, namespace string) {
+			settings, err := settings.LoadFrom(tmpSettingsPath)
+			Expect(err).ToNot(HaveOccurred(), settings)
+
+			settings.User = user
+			settings.Password = password
+			settings.Namespace = namespace
+			err = settings.Save()
+			Expect(err).ToNot(HaveOccurred())
+		}
+
+		BeforeEach(func() {
+			namespace1 = catalog.NewNamespaceName()
+			namespace2 = catalog.NewNamespaceName()
+			env.SetupAndTargetNamespace(namespace1)
+			env.SetupAndTargetNamespace(namespace2)
+
+			service1 = catalog.NewServiceName()
+			service2 = catalog.NewServiceName()
+
+			// create temp settings that we can use to switch users
+			tmpSettingsPath = catalog.NewTmpName("tmpEpinio") + `.yaml`
+			out, err := env.Epinio("", "settings", "update", "--settings-file", tmpSettingsPath)
+			Expect(err).ToNot(HaveOccurred(), out)
+
+			// create users with permissions in different namespaces
+			user1, password1 = env.CreateEpinioUser("user", []string{namespace1})
+			user2, password2 = env.CreateEpinioUser("user", []string{namespace2})
+		})
+
+		AfterEach(func() {
+			By("delete it")
+			deleteServiceFromNamespace(namespace1, service1)
+			deleteServiceFromNamespace(namespace2, service2)
+
+			env.DeleteNamespace(namespace1)
+			env.DeleteNamespace(namespace2)
+
+			// Remove transient settings
+			out, err := proc.Run("", false, "rm", "-f", tmpSettingsPath)
+			Expect(err).ToNot(HaveOccurred(), out)
+
+			env.DeleteEpinioUser(user1)
+			env.DeleteEpinioUser(user2)
+		})
+
+		It("list all services", func() {
+			By("create them in different namespaces")
+			// create service1 in namespace1
+			env.TargetNamespace(namespace1)
+			out, err := env.Epinio("", "service", "create", "mysql-dev", service1)
+			Expect(err).ToNot(HaveOccurred(), out)
+
+			// create service2 in namespace2
+			env.TargetNamespace(namespace2)
+			out, err = env.Epinio("", "service", "create", "mysql-dev", service2)
+			Expect(err).ToNot(HaveOccurred(), out)
+
+			// show all the services (we are admin, good to go)
+			By("show it")
+			out, err = env.Epinio("", "service", "list", "--all")
+			Expect(err).ToNot(HaveOccurred(), out)
+
+			Expect(out).To(MatchRegexp("Listing all Services"))
+
+			Expect(out).To(MatchRegexp(regex.TableRow(namespace1, service1, "mysql-dev", "not-ready")))
+			Expect(out).To(MatchRegexp(regex.TableRow(namespace2, service2, "mysql-dev", "not-ready")))
```

Comment:
I expect these to fail due to the new creation timestamps shown by the list.
If they do not fail I suspect that the new code is missing the timestamps.

Reply:
Confirmed that the new `--all` code in the cli is missing printing of the timestamps.

Reply:
That's right! Fixed